Makefile

```diff
@@ -98,30 +97,16 @@ endif
 .PHONY: ci
 ci: c db-migrate
 ifeq ("$(package)", "all")
-    ifneq ("$(suite)", "new")
-		./hack/bin/cabal-run-integration.sh all
-    endif
-    ifneq ("$(suite)", "old")
-		make c package=integration
-		./hack/bin/cabal-run-integration.sh integration
-    endif
-else
-  ifeq ("$(package)", "integration")
+	 make c
+	./hack/bin/cabal-run-integration.sh all
+	./hack/bin/cabal-run-integration.sh integration
+endif
+ifeq ("$(package)", "integration")
+	make c package=integration
```

Comment:
```suggestion
```
Same here. 

I think probably all these `if`s can go away. It's a bit hard to say for sure. Can you please test the different targets?